### PR TITLE
debuild: add Chinese translation

### PR DESCRIPTION
**Pull request by sandylaw**
Description: add debuild pages in pages.zh/linux.
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).
@ChungZH @KsRyY @whriedplanck

### CODE REVIEW

File: pages.zh/linux/debuild.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# debuild
+
+> 从源代码构建`Debian`软件包的工具。
+> 更多信息： <https://manpages.debian.org/debuild>。
```
Comment (bl-ue) [inline pages.zh/linux/debuild.md:4]:
```suggestion
> 更多信息：<https://manpages.debian.org/debuild>。
```

File: pages.zh/linux/debuild.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# debuild
+
+> 从源代码构建`Debian`软件包的工具。
```
Comment (bl-ue) [inline pages.zh/linux/debuild.md:3]:
```suggestion
> 从源代码构建 `Debian` 软件包的工具。
```

File: pages.zh/linux/debuild.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- 在当前目录中生成软件包：
+
+`debuild`
+
+- 仅构建二进制包：
+
+`debuild -b`
+
+- 生成软件包后，不运行`lintian`（检查常见打包错误）：
```
Comment (bl-ue) [inline pages.zh/linux/debuild.md:14]:
```suggestion
- 生成软件包后，不运行 `lintian`（检查常见打包错误）：
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

File: pages.zh/linux/debuild.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# debuild
+
+> 从源代码构建 `Debian` 软件包的工具。
+> 更多信息：<https://manpages.debian.org/debuild>。
```
Comment (bl-ue) [inline pages.zh/linux/debuild.md:4]:
```suggestion
> 更多信息：<https://manpages.debian.org/debuild>.
```

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me after @bl-ue's comments are addressed.

Thanks :slightly_smiling_face:

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**